### PR TITLE
Add variables for ArchLinux

### PR DESCRIPTION
**Pull request by aruhier**
Add support and variables for ArchLinux

### CODE REVIEW

**Comment by gclough** [comment]
Hi @Anthony25 , is this the only thing required to support ArchLinux?  We don't support `pacman` anywhere, but I must confess that I don't know anything about ArchLinux.  Does it also support yum, or apt?

If this is all that's required, then could you please update the Travis tests to ensure this OS is automatically tested, and the README to show that it's supported.

**Comment by nchudleigh** [comment]
@gclough @Anthony25 Arch is going to be a whole other can of worms, we would need to ensure that postgres was properly supported with repositories and utilities as Arch comes out of the box with the bare minimum.

**Comment by gclough** [comment]
I agree that trying to be everything to everyone is pretty hard, so if this is going to be a big chunk of work then @Anthony25 are you able to update things and make it fully formed?  (Travis, README, etc)  If not, then this will probably get closed. :-(

IMHO we should aim to support all Linux variants that PDGD support:

https://www.postgresql.org/download/

```
Pre-built binary packages are available for a number of different operating systems:

BSD
FreeBSD
OpenBSD
Linux
Red Hat family Linux (including CentOS/Fedora/Scientific/Oracle variants)
Debian GNU/Linux and derivatives
Ubuntu Linux and derivatives
SuSE and OpenSuSE
Other Linux
macOS
Solaris
```

I'm not sure where ArchLinux falls in this grouping, but there does seem to be some support for it:

https://wiki.archlinux.org/index.php/PostgreSQL

**Comment by Anthony25** [comment]
Hi,
Sorry I didn't see your answers. Sure I can make a new Dockerfile for Arch and include it into your tests!

**Comment by Anthony25** [comment]
I added some tests for Arch, plus multiple fixes.

I will complete your readme if it seems okay for you.

File: tasks/configure.yml [hunk 1]
```diff
@@ -91,15 +91,15 @@
...
-- name: PostgreSQL | Check whether the postgres data directory is initialized | RedHat
+- name: PostgreSQL | Check whether the postgres data directory is initialized | RedHat & Arch
   stat:
     path: "{{ postgresql_data_directory }}/PG_VERSION"
-  when: ansible_os_family == "RedHat" and not postgresql_cluster_reset
+  when: ansible_os_family in ["RedHat", "Archlinux"] and not postgresql_cluster_reset
   register: pgdata_dir_initialized
 
-- name: PostgreSQL | Initialize the database | RedHat
+- name: PostgreSQL | Initialize the database | RedHat & Arch
```
[the remark this answers is not in the document]
Comment (Anthony25) [reply tasks/configure.yml]: Yes, exactly

File: .travis.yml [hunk 2]
```diff
@@ -9,26 +9,36 @@ services:
...
-    - IMAGE_NAME="centos:7-builded"
-    - IMAGE_NAME="centos:6-builded"
-    - IMAGE_NAME="fedora:27-builded"
+    - IMAGE_NAME="ubuntu-upstart:14.04" PG_VERSIONS="[9.4, 9.5, 9.6, 10, 11]"
+    - IMAGE_NAME="ubuntu:16.04-builded" PG_VERSIONS="[9.4, 9.5, 9.6, 10, 11]"
+    - IMAGE_NAME="debian:8-builded" PG_VERSIONS="[9.4, 9.5, 9.6, 10, 11]"
+    - IMAGE_NAME="debian:9-builded" PG_VERSIONS="[9.4, 9.5, 9.6, 10, 11]"
+    - IMAGE_NAME="centos:7-builded" PG_VERSIONS="[9.4, 9.5, 9.6, 10, 11]"
+    - IMAGE_NAME="centos:6-builded" PG_VERSIONS="[9.4, 9.5, 9.6, 10, 11]"
+    - IMAGE_NAME="archlinux:builded" PG_VERSIONS="[11]"
```
[the remark this answers is not in the document]
Comment (Anthony25) [reply .travis.yml]: As it's a rolling release, it only offers the last version, so yes you can only have the 11 one for postgres from the official repositories.

**Comment by gclough** [comment]
Thanks for answering the questions @Anthony25 .  I'm waiting on some of the maintainers with longer standing than I to weigh in and give approval.

**Pull request closed without merging**